apps/synapse-portal-framework/src/pages/FileEntityPage/FileEntityPageHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Container, Typography } from '@mui/material'
+import { EntityBundle } from '@sage-bionetworks/synapse-types'
+import { IconSvg } from 'synapse-react-client'
+import { FileEntity } from '@sage-bionetworks/synapse-types'
+import CitationPopover from 'synapse-react-client/components/CitationPopover'
+
+const FileEntityPageHeader = ({ data }: { data: EntityBundle }) => {
+  const entity = data.entity as FileEntity
+
+  return (
+    <Box
```

Comment:
Would it make sense to use the `HeaderCard` component here? Or could we extract some common styles/components? I think all of the table logic has been extracted out of HeaderCard, so I would think you could just use it here